### ISSUE TITLE
Rename x-resources and move it inside preferences

### ISSUE THREAD

**Issue by larsenwork** [fontforge/fontforge#2260]
I will add a mockup here soon (I just needed to reference this from another issue) :)

**Comment by larsenwork** [comment]
Great idea to make a milestone - makes it easier to structure and document things :+1:

**Comment by pathumego** [comment]
@larsenwork :+1:  @lpsandaruwan  is planning out his work here : https://docs.google.com/document/d/1_oWAitHWJadgmUGTt1M7oMBXfElK091cEmSsDaF3saA/edit

**Comment by pathumego** [comment]
@larsenwork any ideas on the mock up? I think the simple thing to do for now is to include link to Theme Editor inside prefs?

**Comment by larsenwork** [comment]
@pathumego yes, see https://github.com/fontforge/fontforge/issues/2321#issue-85041439

**Comment by davelab6** [comment]
I don't really understand the intention of this issue

**Comment by larsenwork** [comment]
@davelab6 to rename x-resources to theme editor and move it from the file menu and inside preferences

**Comment by davelab6** [comment]
I don't think combining themes and preferences makes sense... but you all should do what you think is best :)

**Comment by larsenwork** [comment]
I don't know where else it should be - most other apps I've used have appearance/themes inside the general settings e.g. xCode:
![screen shot 2015-06-07 at 20 58 26](https://cloud.githubusercontent.com/assets/3859425/8025458/fd791cb2-0d57-11e5-8a9b-56194c82b13f.png)

**Comment by pathumego** [comment]
Themes are preferences. :) from is level to apps ui  options are in settings.

**Comment by larsenwork** [comment]
@pathumego I'm a bit confused about your time zone - isn't it way past midnight?

**Comment by pathumego** [comment]
@larsenwork im also confused about it :) Im used to working in the evenings unless I have meetings during the day :/ not healthy @lpsandaruwan is waiting on the mockup to add this change. But Im not sure how we could add this into Prefs

**Comment by larsenwork** [comment]
I can mock it up but I've written it out here: https://github.com/fontforge/fontforge/issues/2321

**Comment by pathumego** [comment]
Done in new-menus
